### PR TITLE
Feature/setup

### PR DESCRIPTION
**Pull request by jdmartinez1062**
Completed React app setup using npm. Deployed the app to heroku.

### CODE REVIEW

**Review by Forison — CHANGES_REQUESTED** [review]
#### Invalid❕❕

##### You need to enable linter checks on this or before it can be review. This would be marked invalid. Kindly address the issue and submit a re-review

#### Happy coding
Addo Forison

File: README.md [hunk 1]
```diff
@@ -1 +1,70 @@
-# Calculator
\ No newline at end of file
+# Getting Started with Create React App
+
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
```
Comment (bellom) [inline README.md:3]: **Kindly update your `README` file**
**Let's make your repo look more professional by updating your `README` file**.
Checkout out the below link for help, feel free to follow this [Microverse template](https://github.com/microverseinc/readme-template) from pathwright

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi @jdmartinez1062  :clap: 

### Awesome job so far!! :+1: 

#### You have done wonderfully well, you passed `almost` all the checklist for this project :+1: just a few issues left to work on............ :slightly_smiling_face: 

# Status: Changes Required! ♻️ 

**Kindly check out my comments below;**

### ♻️ Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Review by mikenath223 — CHANGES_REQUESTED** [review]
### Hello @jdmartinez1062 👋

**You've done pretty well here fixing a few issues raised in the previous review. 👌**

However, let's make a couple of changes.

- Please refer to comments on the code.

### **STATUS: CHANGES REQUESTED** 👷‍♂️

Kindly fix these issues and request another review👍
Feel free to contact me on slack(michgolden) if you need further clarification.

**HAPPY CODING 🎉💻🎉**

**Michgolden Ukeje (CR)**

File: README.md [hunk 2]
```diff
@@ -1 +1,131 @@
...
+- [Contributing](#contributing)
+- [Show your support](#show-your-support)
+- [Acknowledgments](#acknowledgments)
+- [License](#license)
+
+<!-- ABOUT THE PROJECT -->
+
+## About The Project
+
+
```
Comment (mikenath223) [inline README.md:51]: - [ ] Kindly deploy the project online and include the live link in the readme.

**Review by Dipeshtwis — APPROVED** [review]
# Status: Approved ✔️

## Hi @jdmartinez1062,

Good job on implementing the suggested changes. Well Done.
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

**You may close this Pull Request and merge the changes**

  
**Keep being awesome devs and good luck with your next milestone!**
---------

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**_Dipesh Kumar_**

**Pull request closed without merging**